packages/core-app-api/src/lib/AuthSessionManager/AuthSessionStore.ts

```diff
@@ -27,6 +28,8 @@ type Options<T> = {
   manager: MutableSessionManager<T>;
   /** Storage key to use to store sessions */
   storageKey: string;
+  /** The schema used to validate the stored data */
+  schema: ZodSchema<T>;
```

Comment:
We are tying ourselves to Zod by exposing the ZodSchema type here. Are we fine with that?

Reply:
This class is actually not exported!

Reply:
Then I feel a lot better 👍